app/[lang]/page.tsx

```diff
@@ -3,10 +3,12 @@ import React from 'react';
 import LiatoshynskyOffice from '~/components/Liatoshynsky-office/LiatoshynskyOffice';
 
 import FoundationFounders from '~/shared/components/FoundationFounders/FoundationFounders';
+import OurMission from '~/shared/components/our-mission/OurMission';
```

Comment:
```suggestion
import FoundationFounders from '~/components/FoundationFounders/FoundationFounders';
import OurMission from '~/components/our-mission/OurMission';
```

We have alias for components, so no need to write share no more

Reply:
Ok, thanks, gonna fix that.

Reply:
Done.